authors/settings.py

```diff
@@ -134,10 +134,8 @@
 AUTH_USER_MODEL = 'authentication.User'
 
 REST_FRAMEWORK = {
-    'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
```

Comment:
Is there a particular reason why this exception handler was removed?

Reply:
This is an issue that has been brought about by changes while rebasing. Let me respond in kind.